tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -221,9 +221,11 @@ Status GetEngineInfo(const Graph* g,
       }
       if (edge->IsControlEdge()) {
         // Control output.
-        info->connections.emplace_back(output_node->name(), output_node->id(),
```

Comment:
Shall we have a helper function, say ShallKeepConrolEdgeFrom(Node* input_node), for here and the place that handle node? 
According to the change for the input node, it seems that the purpose of removing such edges  is to allow removing unused constants. But how does this expose unused constants is something I don't understand, still trying to find out about this.

Reply:
I have added the ShallKeepControlEdgeFrom helper function.